drivers/adc/ad9656/ad9656.c

```diff
@@ -150,15 +176,15 @@ int32_t ad9656_setup(struct ad9656_dev **device,
 	if (ret != SUCCESS)
 		return ret;
 
+	ad9656_spi_write(dev, AD9656_SPI_CONFIG, 0x3C);	/* RESET */
```

Comment:
modification not related to commit, probably it should go in the first one.

Reply:
I just moved the reset instruction for the AD9656 chip to be the first instruction that is executed on this chip in the setup process